### ISSUE TITLE
cached results are shown for references/citation links

### ISSUE THREAD

**Issue by aaccomazzi** [adsabs/bumblebee#1731]
## Steps to Reproduce

1. do any search, e.g. https://qa.adsabs.harvard.edu/#search/p_=0&q=%20author%3A%22kurtz%2C%20m%22&sort=date%20desc%2C%20bibcode%20desc
2. click on the first result, you'll get to https://qa.adsabs.harvard.edu/#abs/2019arXiv190300297K/abstract
3. go back to the list of results, select the references link for item #6 
4. you will get to a page with an error message (no references found as of today) or the references for the previous paper.

Basically selecting references/citations from a result page shows the previous (cached) list.
